building/config.json

```diff
@@ -387,6 +387,12 @@
     "path": "building/tracks/stories/tuples.santas-helper.md",
     "title": "Santa's Helper"
   },
+  {
+    "uuid": "bbb766b6-4b28-4a19-bf57-48495b465c86",
+    "slug": "tracks/ci/workflow-templates",
+    "path": "building/tracks/ci/workflow-templates.md",
+    "title": "Workflow Templates"
+  },
```

Comment:
We need a CI doc as well for this to work.
This should live at `tracks/ci/README.md` and contain an overview of the CI of tracks.

It could have some basic text like "At Exercism, we use GitHub actions to handle our continuous intergration (CI) requirements. This includes running tests, formatting things, and deploying things."